### PR TITLE
Support a Gaussian blur effect field that sometimes shows up.

### PR DESCRIPTION
**Pull request by simeoncran**
And polish the yaml serializer a bit.

### CODE REVIEW

File: source/YamlData/YamlScalar.cs [hunk 1]
```diff
@@ -28,7 +29,7 @@ internal YamlScalar(object value, string presentation)
             var escapedValue = value;
             if (escapedValue is null)
```
Comment (simeoncran) [inline source/YamlData/YamlScalar.cs:30]: value is not nullable. Why is escapedValue nullable?

File: source/LottieReader/Serialization/Effects.cs [hunk 2]
```diff
@@ -101,12 +102,19 @@ GaussianBlurEffect ReadGaussianBlurEffect(in LottieJsonObjectElement obj, string
                         repeatEdgePixels = ReadAnimatableBool(value) ?? throw ParseFailure();
                         break;
 
+                    case 3:
+                        // Optional parameter. Parse it is an animatable because it has the
```
Comment (derobinsMSFT) [inline source/LottieReader/Serialization/Effects.cs:106]: >is [](start = 56, length = 2)

"as"

File: source/LottieReader/Serialization/Effects.cs [hunk 3]
```diff
@@ -101,12 +102,19 @@ GaussianBlurEffect ReadGaussianBlurEffect(in LottieJsonObjectElement obj, string
...
+                        // animatable format, however it would be very strange if the value
+                        // was actually animated so convert it to non-animated bool;
```
Comment (derobinsMSFT) [inline source/LottieReader/Serialization/Effects.cs:108]: >; [](start = 83, length = 1)

Nit: period.

**Review by derobinsMSFT — APPROVED** [review]
:shipit:

**Pull request merged**